.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Windows CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  SBT_OPTS: '-Xmx6g -Xms1g -Xss4m'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [ '8' ]
+
+    runs-on: windows-latest
+
+    steps:
+    - name: Set up git to use LF
+      run: |
+        git config --global core.autocrlf false
+        git config --global core.eol lf
+    - uses: actions/checkout@v3
+      with:
+        # fetch the Git tags for sbt-dynver and sbt-version-policy to work correctly
+        fetch-depth: 0
+    - name: Set up JDK ${{ matrix.java }}
+      uses: coursier/setup-action@v1
+      with:
+        jvm: temurin:1.${{ matrix.java }}
+        apps: sbt
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '24.x'
```

Comment:
Is there a reason we are upgrading node with this change?

Reply:
The reason is "why not, while we're here?". The LInux CI currently uses 23.x (we upgraded for the Wasm support) but we should also upgrade it to 24 relatively soon.